backend/services/syson-diagram-services/src/main/java/org/eclipse/syson/diagram/services/DiagramQueryElementService.java

```diff
@@ -193,6 +198,64 @@ public String getGraphicalParentId(DiagramContext diagramContext, Node node) {
         return parentId;
     }
 
+    /**
+     * Get the semantic element of the graphical parent of the given node. If located at the root of the diagram, return the semantic element of the diagram (most likely the {@link ViewUsage)}.
+     *
+     * @param node
+     *         the node to test
```

Comment:
the node to test => the node for which we want to find the semantic element corresponding to its graphical parent node